exception/Exception.js

```diff
@@ -15,37 +15,32 @@ import {isString} from 'lodash';
  */
 export class Exception {
 
+    /**
+     * Create and get back a Javascript Error object
+     * @param {(Object)} cfg - properties to add to the Error object
```

Comment:
Did we mean this to be `{(Object|string)}`?

Reply:
Yes. I've corrected this in the develop branch.

Reply:
Thanks much!